source/documentation/deploying_services/postgres.md

```diff
@@ -170,19 +170,19 @@ Note that data restore will not be available in the event of an RDS outage affec
 
 For more details about how the RDS backup system works, see [Amazon's DB Instance Backups documentation](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.BackingUpAndRestoringAmazonRDSInstances.html) [external page].
 
-### Restoring a PostgreSQL service snapshot
+### Restoring a PostgreSQL service snapshot (experimental)
 
 As a tenant, you can create a copy of any existing PostgreSQL service instance using the latest snapshot of the RDS instance.
```

Comment:
This wasn't something that was changed as part of this pull request, but I can do it.